com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -467,19 +493,22 @@ protected virtual void SetRotation()
         /// </summary>
         private void RegisterPlacementAction()
         {
-            InputAction placementAction = GetInputActionFromReference(placementActionReference);
-            if (placementAction == null)
+            foreach(InputAction placementAction in placementActions)
             {
-                Debug.Log("Failed to register the placement action, the action reference was null or contained no action.");
-                return;
+                if (placementAction == null)
+                {
+                    Debug.Log("Failed to register the placement action, the action reference was null or contained no action.");
+                    return;
+                }
+                placementAction.performed += StopPlacementAction;
             }
-            placementAction.performed += StopPlacement;
+            lastActions = placementActions;
         }
 
         /// <summary>
         /// Registers the event which performs pickup.
         /// </summary>
-        private void RegisterPickupAction()
+        private void RegisterPickupEvent()
         {
             if (interactable == null)
```

Comment:
since `interactable` is a required component, a null check doesn't make sense. This should be exceptional and cause an exception.